lib/watch.js

```diff
@@ -132,13 +133,25 @@ const start = async (env, directAssetsUpload) => {
   const push = directAssetsUpload ? pushFileDirectAssets : pushFile;
   if (directAssetsUpload) await fetchDirectUploadData(gateway);
 
+  let liveReloadServer;
+  if (liveReload) {
+    liveReloadServer = livereload.createServer();
```

Comment:
IMO we should here:

a) observe only `app`, `modules`
b) observe only extensions we are interested in ( https://github.com/mdyd-dev/pos-cli/blob/331b01a0e046b6665a10084515374b8644fbf019/lib/watch-files-extensions.js )
c)